.travis.yml

```diff
@@ -144,6 +144,8 @@ script:
     fi
   # different builds with CMake
   - if [ "$BUILD_MODE" == "cmake" ]; then
+      # Use early version of cmake
+      sudo -H $PIP install cmake==3.6.3
```

Comment:
> So this #812 immediately has same issue as #807. Perhaps sudo is to blame. But where is the log?

You mentioned `sudo` and I realised it's syntax error! In case of list statement, underneath statements inside if-else  needs semicolon! So it should be `sudo -H $PIP install cmake==3.6.3;`

Reply:
If there is syntax error then travis doesn't able to parse yaml file and hence it doesn't do anything (but it's true that it's very confusing!)